src/lib/tasks/run-callbacks.ts

```diff
@@ -109,13 +109,8 @@ export function runCallbacks(
     // Append the current time to the arrival time value.
     spriteView.TransitionTimeMs += currentTimeMs;
 
-    // Make sure that the draw Ts range includes the current transition time
-    // plus a buffer to account for time taken by work tasks. Without the
-    // buffer, it can happen that the last drawn frame does not include the
-    // final resting state of the Sprite, especially when the user-specified
-    // transition time is near or below one frame (about 17ms).
-    coordinator.toDrawTsRange.expandToInclude(
-        spriteView.TransitionTimeMs + coordinator.workScheduler.maxWorkTimeMs);
```

Comment:
This solution only worked when `maxWorkTimeMs` was greater than or equal to the time between animation frames. Implementing constrained render tests revealed that this would not guarantee an additional draw call in all cases. Since this was not always effective, it's better to implement something that is.

Note the addition of `queueDraw()` to the Texture sync task. It is always the case that after a Texture sync, there needs to be a draw for the rendered pixels to show the changed values. Texture sync should always have queued a draw call, and now it does.

Before, with the hack solution above, we would get lucky that the added time happened to be long enough. Now, there will always be at least one more draw after a Texture sync.